src/EnergyPlus/FileSystem.hh

```diff
@@ -85,17 +88,45 @@ namespace FileSystem {
 
     extern std::string const exeExtension;
 
-    enum class InputFileType
+    enum class FileTypes
     {
+        Unknown,
```

Comment:
We generally call this `Unassigned`, and also set it to -1.

Reply:
I don't think `Unassigned` makes sense for a file format we don't know, while `Unknown` does make more sense to me. At the end of the day, it doesn't really matter since no user sees this enum.

Reply:
`Unknown` is fine. `-1` is more important, because that is what getEnumerationValue will return if it doesn't find the value.